storage/cache/subtree_cache.go

```diff
@@ -58,6 +58,8 @@ const (
 	depthQuantum = 8
 	// logStrataDepth is the strata that must be used for all log subtrees.
 	logStrataDepth = 8
+	// maxLogDepth is the number of bits in a log path.
```

Comment:
I'll add an issue for it. #754
At present, many of the log processing algorithms us `int64` for the leaf index, which basically caps the log at 64 levels anyways.  In an upcoming PR, I plan to create a consistent interface for `NodeID` which could help with this. Switching onto it would be an optional way to get around this constraint.